Assets/Plugins/LeapMotion/Core/Examples/Example Assets/InertiaPostProcessProvider.cs

```diff
@@ -126,17 +126,17 @@ public class InertiaPostProcessProvider : PostProcessProvider {
     private void integratePose(ref Pose curPose, ref Pose prevPose,
                                Pose targetPose, float deltaTime) {
       // Calculate motion from prevPose to curPose.
-      var deltaPose = curPose.inverse * prevPose; // prevPose in curPose's local space.
+      var deltaPose = PoseExtensions.mul(curPose.inverse(), prevPose); // prevPose in curPose's local space.
```

Comment:
Consider infix/chaining calls instead of manually referencing PoseExtensions (they are extension methods, right?):
`curPose.inverse().mul(prevPose)`

Reply:
(also applies to the other PoseExtensions calls in this file and others)